packages/server/src/helpers/calculateMaxMintableForMember.ts

```diff
@@ -1,10 +1,21 @@
 import { DocumentSnapshot } from "@google-cloud/firestore";
 import Big from "big.js";
 
-const RAHA_UBI_WEEKLY_RATE = 10;
+const RAHA_UBI_WEEKLY_RATE = new Big(10);
+const MAX_WEEKS_ACCRUE = new Big(4);
 const MILLISECONDS_PER_WEEK = 1000 * 60 * 60 * 24 * 7;
-// TODO enable this on the 11/15/2018.
-export const MINT_CAP = 40;
+export const MINT_CAP = RAHA_UBI_WEEKLY_RATE.times(MAX_WEEKS_ACCRUE);
+
+// Set to midnight on Nov 16th, 2018 UTC
+const MINT_CAP_TRANSITION_DATE_UTC = Date.UTC(2018, 10, 16);
```

Comment:
nit: I think moving these consts to shared would be nice

Reply:
agreed, I'll do that in a separate PR